trio/_highlevel_open_unix_stream.py

```diff
@@ -36,12 +36,13 @@ def close_on_error(obj):
     Raises:
       OSError: If the socket file could not be connected to.
       RuntimeError: If AF_UNIX sockets are not supported.
+      TypeError: if filename is not str or bytes.
```

Comment:
I don't think we should mention this in the documentation. It's a general rule that pretty much any Python function can raise `TypeError` if you give it the wrong argument types, and mentioning it explicitly here distracts from more relevant information.

Reply:
Ok for `TypeError` I removed it